Directory.Build.props

```diff
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <VersionMajor Condition="'$(VersionMajor)' == ''">1</VersionMajor>
         <VersionMinor Condition="'$(VersionMinor)' == ''">0</VersionMinor>
-        <VersionPatch Condition="'$(VersionPatch)' == ''">1</VersionPatch>
+        <VersionPatch Condition="'$(VersionPatch)' == ''">2</VersionPatch>
```

Comment:
> patch version bump, assuming this project follows semver

Yeah, this project follows semver.

The version-numbers in the MsBuild-file can be overwritten with environment variables (like every MsBuild-property). For the build / deploy I use https://circleci.com and the tag-name (like v1.0.2) sets the version-number -> https://github.com/gfoidl/trx2junit/blob/504004e0aecda39bfbe64ff7a60366bb13bf0bee/build.sh#L78-L83

So the values in the MsBuild-file are more or less just for local builds / deploys.

I would use version 1.1.0, so an update to the minor-number, as this PR together with the fix for https://github.com/gfoidl/trx2junit/issues/14 would become a new version, rather than a bug-fix.

"Intermediate" packages are released to [MyGet](https://www.myget.org/feed/gfoidl/package/nuget/trx2junit), before they get to [NuGet](https://www.nuget.org/packages/trx2junit/).